src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -43,6 +46,7 @@ def load_arguments(self, _):
         c.argument('onboarding_timeout', options_list=['--onboarding-timeout'], arg_group='Timeout', help='Time required (in seconds) for the arc-agent pods to be installed on the kubernetes cluster. Override this value if the hardware/network constraints on your cluster requires more time for installing the arc-agent pods.')
         c.argument('no_wait', options_list=['--no-wait'], arg_group='Timeout', help="Do not wait for the long-running operation to finish.")
         c.argument('correlation_id', options_list=['--correlation-id'], help='A guid that is used to internally track the source of cluster onboarding. Please do not modify it unless advised', validator=override_client_request_id_header)
+        c.argument('container_log_path', options_list=['--container-log-path'], help='Override the default container log path to enable fluent-bit logging')
```

Comment:
In fact, `--container-log-path` is the default option name for parameter `container_log_path`, so there is no need to define `options_list=['--container-log-path']` repeatedly

Reply:
fixed